libsel4/include/sel4/bootinfo_types.h

```diff
@@ -70,9 +66,7 @@ typedef struct seL4_BootInfo {
     seL4_SlotRegion   extraBIPages;    /* caps for any pages used to back the additional bootinfo information */
     seL4_Word         initThreadCNodeSizeBits; /* initial thread's root CNode size (2^n slots) */
     seL4_Domain       initThreadDomain; /* Initial thread's domain ID */
-#ifdef CONFIG_KERNEL_MCS
     seL4_SlotRegion   schedcontrol; /* Caps to sched_control for each node */
```

Comment:
If we keep this slot unconditionally, we should probably add a comment what we expect to see here in the non-MCS case.

Reply:
I wasn't really sure if it should be left in our not. Should we just leave this as conditional?

Reply:
I just put the ifdef back in.